Test/alpha/conanfile.py

```diff
@@ -66,16 +68,8 @@ def package(self):
 		cmake.install()
 
 	def package_info(self):
-		self.cpp_info.set_property("cmake_file_name", self.name)
-		self.cpp_info.set_property("cmake_target_name", f"{self.name}::{self.name}")
-
 		for compname, comp in self._alpha_component_tree.items():
-			conan_component = f"{self.name}_{compname.lower()}"
-			requires = [f"{self.name}_{dependency.lower()}" for dependency in comp.dependencies] + comp.external_dependencies
-			self.cpp_info.components[conan_component].set_property("cmake_target_name", f"{self.name}::{compname}")
-			self.cpp_info.components[conan_component].set_property("cmake_file_name", compname)
```

Comment:
Broken, the ``cmake_file_name`` cannot be per-component.
In general, no need to define these properties, those are only for ConanCenter recipes.